src/main.rs

```diff
@@ -108,6 +108,23 @@ enum Command {
     #[structopt(name = "docs", about = "Render HTML documentation for a project")]
     Docs(Docs),
 
+    #[structopt(name = "init", about = "Create a new project in an existing directory")]
+    Init {
```

Comment:
I think we'll also want to optionally take the name of the project because we write this `gleam.toml` and the user may want not to use the name of the directory.

```toml
name = "the_project_name"
```